README.md

```diff
@@ -1,19 +1,317 @@
+      "licenceType": "DriverOfTaxisAndPrivateHires",
+      "licenceTimeTrading": "ZeroToTwoYears",
+      "licenceValidityPeriod": "UpToThreeYears"
+   },
+   "taxDetails": { 
+      "nino": "AB123456C",
+      "sautr": "1234567890",
+      "taxSituation":"SA",
+      "saIncomeDeclared": "Yes",
+      "saStatusResponse": {
+         "sautr": "1234567890",
+         "taxYear": 2020,
+         "status": "ReturnFound"
+      },
+      "relevantIncomeTaxYear": 2020,
+      "correctiveAction": "DormantAccountReactivated"
+   },
+   "taxCheckStartDateTime": "2021-12-02T14:56:11.75Z[Europe/London]",
+   "source":"Stride",
+   "type":"Individual"
+}
+```
+Example response body:
+```json
+{
+   "taxCheckData" : {
+      "applicantDetails": {
+         "ggCredId": "credId",
+         "crn": "1234567890",
+         "companyName": "Test Company"
+      },
+      "licenceDetails": {
+         "licenceType": "OperatorOfPrivateHireVehicles",
+         "licenceTimeTrading": "TwoToFourYears",
+         "licenceValidityPeriod": "UpToOneYear"
+      },
+      "taxDetails": {
+         "desCTUTR" :"1111111111",
+         "userSuppliedCTUTR": "1111111111",
+         "ctIncomeDeclared": "No", 
+         "ctStatus": {
+            "ctutr":  "1234567890",
+            "startDate":  "20201001",
+            "endDate":  "20193009",
+            "latestAccountingPeriod": {
+               "startDate": "20200131",
+               "endDate": "20200524",
+               "ctStatus": "NoticeToFileIssued"
+            }
+         }, 
+         "recentlyStaredTrading": "No",
+         "chargeableForCT": "Yes",
+         "correctiveAction": "Other"
+      },
+      "taxCheckStartDateTime": "2021-12-02T14:56:11.75Z[Europe/London]",
+      "source": "Digital",
+      "type": "Company"
+   },
+   "taxCheckCode": "TLAR4D6HB",
+   "expiresAfter": "2022-04-01",
+   "createDate": "2021-12-02T14:56:49.618Z[Europe/London]",
+   "isExtracted": false
+}
+```
+N.B. the combinations of answers in the request and response bodies above do not correspond to actual
+real business scenarios. A value for each field (even if optional) has been provided for illustrative 
+purposes.
+
+
+### `POST /hec/match-tax-check`
+Check the details given in the request to see if they match an existing tax check in the database. A
+`200` (OK) response is returned if a match result can be given. The example request body shows an example
+for an individual and the example response body includes an example for a company. 
+
+Example request body:
+```json
+{
+   "taxCheckCode": "222RRR888",
+   "licenceType": "ScrapMetalDealerSite",
+   "verifier": {
+      "dateofbirth": "19221201"
+   }
+}
+```
+
+Example response body:
+```json
+{
+   "matchRequest": {
+      "taxCheckCode": "7PCR3ANY2",
+      "licenceType": "ScrapMetalMobileCollector",
+      "verifier": {
+         "crn": "1234567890"
+      }
+   },
+   "dateTimeChecked": "2021-12-02T15:25:57.728Z[Europe/London]",
+   "status": "NoMatch"
+}
+```
+The req
```

Comment:
I think this is typo

Reply:
whoops! Good spot -  just removed it [here](https://github.com/hmrc/hec/pull/53/commits/5297f926e288d6c5460ad819cee8c5364321a82a)